onnxruntime/core/optimizer/dynamic_quantize_matmul_fusion.cc

```diff
@@ -94,34 +138,87 @@ Status DynamicQuantizeMatMulFusion::ApplyImpl(Graph& graph, bool& modified, int
       continue;
     }
 
-    std::vector<NodeArg*> input_defs{dql_node_left.MutableInputDefs()[0],
-                                     matmulinteger_node.MutableInputDefs()[1],
-                                     mul_node_right.MutableInputDefs()[1]};
+    // Find bias node
+    Node* add_node = nullptr;
+    // const Node* add_node = FindBiasNode(graph, mul_node, ;
+    if (optimizer_utils::CheckOutputEdges(graph, mul_node, 1)) {
+      const Node* tmp_add_node = graph_utils::FirstChildByType(mul_node, "Add");
```

Comment:
If this fails should there be a 'continue'?

Reply:
yes, if it failes, then the bias will be null. 

---
In reply to: [448807810](https://github.com/microsoft/onnxruntime/pull/4350#discussion_r448807810) [](ancestors = 448807810)